modules/network-observability-RTT-overview.adoc

```diff
@@ -5,24 +5,24 @@
 :_mod-docs-content-type: CONCEPT
 [id="network-observability-RTT-overview_{context}"]
 = Round-Trip Time
-You can use TCP handshake Round-Trip Time (RTT) to analyze network flows. You can use RTT captured from the `fentry/tcp_rcv_established` eBPF hookpoint to read SRTT from the TCP socket to help with the following:
+You can use TCP smoothed Round-Trip Time (sRTT) to analyze network flow latencies. You can use RTT captured from the `fentry/tcp_rcv_established` eBPF hookpoint to read sRTT from the TCP socket to help with the following:
```

Comment:
Do we need to spell out RTT in the second sentence?

Reply:
I am confused by the reference to sRTT here, but not throughout the module.

Reply:
Good observations. So the dashboard info in the Overview is still "Round-Trip Time" wording:
>  By default, when RTT is enabled, you can see the following TCP RTT metrics represented in the Overview:
Top X 90th percentile TCP Round Trip Time with overall
Top X average TCP Round Trip Time with overall
Bottom X minimum TCP Round Trip Time with overall
Other RTT panels can be added in Manage panels:
Top X maximum TCP Round Trip Time with overall
Top X 99th percentile TCP Round Trip Time with overall

I double checked the other mentions with my SME @msherif1234 and he says it is OK to refer to this feature as RTT its just that the kind of RTT in the [API](https://github.com/netobserv/network-observability-operator/blob/1ac01425ea97e8da2efade1b0a795b7ada17947a/apis/flowcollector/v1beta2/flowcollector_types.go#L329) is changing (I think I'm paraphrasing him correctly 😅 )